### PR TITLE
Updated UI

### PR DESCRIPTION
**Pull request by melwinlobo18**

### CODE REVIEW

**Comment by rumaan** [comment]
Hey thaks for the PR could you please add some screenshots of the new UI?:smiley:

**Comment by melwinlobo18** [comment]
Should I edit the readme file or include them here?

**Comment by rumaan** [comment]
Check out #21

File: lib/ui/home/home.dart [hunk 1]
```diff
@@ -25,55 +22,72 @@ class _HomePageState extends State<HomePage> {
...
+                return ListView.builder(
+                    itemCount: snapshot.data.length + 1,
+                    itemBuilder: (context, index) {
+                      if (index == snapshot.data.length) {
+                        return _buildLoading();
+                      }
+                      return ApodListItem(
+                        item: snapshot.data[index],
+                      );
+                    });
```
[the remark this answers is not in the document]
Comment (melwinlobo18) [reply lib/ui/home/home.dart]: I have replaced `CustomScrollView` by `NestedScrollView`. So I am using `SliverAppBar` as the head and `ListView.builder` as body.

[the remark this answers is not in the document]
Comment (rumaan) [reply lib/ui/home/home.dart]: Cool

**Pull request merged**